drivers/remote/messages.go

```diff
@@ -93,6 +105,32 @@ func (r *createEndpointResponse) parseInterfaces() ([]*iface, error) {
 	return ifaces, nil
 }
 
+func (r *joinResponse) parseStaticRoutes() ([]*types.StaticRoute, error) {
+	var (
+		routes = make([]*types.StaticRoute, len(r.StaticRoutes))
+	)
+	for i, inRoute := range r.StaticRoutes {
+		var err error
```

Comment:
You don't need to declare this as a variable. Use type inference in the code below where you are using `err`